### ISSUE TITLE
Switching from between rooms causes frontend to become unresponsive

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/spreed#2894]
## Steps to reproduce
I can only reproduce this with a room that is shared via a public link and there is one guest user in it (room A)

1. Be in room A
2. Change to a different room
3. Switch back to room A

### Expected behaviour
Ui stays responsive

### Actual behaviour
UI seems broken, changing rooms doesn't work anymore. This does only happen with that one specific room described above. As it happens on our instance, i can give you the room token if needed for further debugging.

## Browser

Chromium

### Browser log

```
TypeError: Cannot read property 'key' of undefined
    at ir (vue.runtime.esm.js:5817)
    at vue.runtime.esm.js:6219
    at D (vue.runtime.esm.js:6319)
    at vue.runtime.esm.js:6193
    at D (vue.runtime.esm.js:6319)
    at a.__patch__ (vue.runtime.esm.js:6482)
    at a.e._update (vue.runtime.esm.js:3948)
    at a.r (vue.runtime.esm.js:4066)
    at fn.get (vue.runtime.esm.js:4479)
    at fn.run (vue.runtime.esm.js:4554)
ze @ vue.runtime.esm.js:1888
Ue @ vue.runtime.esm.js:1879
Ge @ vue.runtime.esm.js:1839
(anonymous) @ vue.runtime.esm.js:1982
$e @ vue.runtime.esm.js:1906
Promise.then (async)
We @ vue.runtime.esm.js:1933
tt @ vue.runtime.esm.js:1990
(anonymous) @ vue.runtime.esm.js:4402
fn.update @ vue.runtime.esm.js:4544
le.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
updateToken @ tokenStore.js:45
(anonymous) @ vuex.esm.js:740
(anonymous) @ vuex.esm.js:392
(anonymous) @ vuex.esm.js:391
u._withCommit @ vuex.esm.js:522
u.commit @ vuex.esm.js:390
commit @ vuex.esm.js:335
updateToken @ tokenStore.js:70
(anonymous) @ vuex.esm.js:747
u.dispatch @ vuex.esm.js:438
dispatch @ vuex.esm.js:332
e @ App.vue:237
(anonymous) @ App.vue:265
l @ vue-router.esm.js:2154
r @ vue-router.esm.js:1880
Yt @ vue-router.esm.js:1888
Qt.confirmTransition @ vue-router.esm.js:2181
Qt.transitionTo @ vue-router.esm.js:2068
t.push @ vue-router.esm.js:2399
Kt.push @ vue-router.esm.js:2814
m @ vue-router.esm.js:1097
He @ vue.runtime.esm.js:1854
n @ vue.runtime.esm.js:2175
Dr.o._wrapper @ vue.runtime.esm.js:6917
```

Messages after that error:

![image](https://user-images.githubusercontent.com/3404133/73742112-f9f62980-474b-11ea-9135-f95723ddb9f5.png)

**Comment by nickvergessen** [comment]
It actually happens when you rejoin a conversation. Not sure how yet, but it happens for me with ivans one-to-one chat e.g.

**Comment by nickvergessen** [comment]
Should be fixed with #2870
